Cargo.toml

```diff
@@ -56,7 +57,7 @@ log = ["chrono"]
 sql = ["mysql", "serde", "serde_json", "once_cell", "dashmap", "jobs"]
 
 # non-default features
-hash = ["md-5", "sha-1", "sha2", "hex", "twox-hash", "const-random"]
+hash = ["base64", "const-random", "md-5", "hex", "sha-1", "sha2",  "twox-hash", "twox-hash"]
```

Comment:
this is accidental, right?
```suggestion
hash = ["base64", "const-random", "md-5", "hex", "sha-1", "sha2",  "twox-hash"]
```

Reply:
no, the person who PR'd it forgot to add it in #60, lol

Reply:
oh fuck i'm an idiot

Reply:
But I can see it 😕